.github/workflows/test_setup_and_docs.yml

```diff
@@ -12,6 +12,10 @@ jobs:
       - name: Checkout repository
         uses: actions/checkout@v3
 
+      - name: Add PPA
+        run: |
+          sudo add-apt-repository ppa:git-core/ppa -y
```

Comment:
Good and simple approach to solve this CI problem!!! 😄

However, I'm going to have to be a little more nitpicky, as I consider the bar higher for CI changes 😬.

3 things:

1. Please, add a succinct comment above the job to explain the reasoning behind adding this PPA and change the name of the job to something more specific (`Add PPA` is too broad). If you can achieve a self-explanatory job name, you can drop the comment part.
2. Just to keep the commit history consistent (no commit fails in the CI) move this commit to be the first in the series, as the CI bug you detected is independent from the current first commit.
3. I know that being a CI bug, you are limited in term of fixes, but is this fix "weak" (i.e., it may stop working in the future)? @rodrigosiqueira, what do you think?

Reply:
When we migrate to Sourcehut, I think this will not be an issue. However, for now, it is acceptable to use this PPA.

Reply:
> Good and simple approach to solve this CI problem!!! 😄
> 
> However, I'm going to have to be a little more nitpicky, as I consider the bar higher for CI changes 😬.
> 
> 3 things:
> 
> 1. Please, add a succinct comment above the job to explain the reasoning behind adding this PPA and change the name of the job to something more specific (`Add PPA` is too broad). If you can achieve a self-explanatory job name, you can drop the comment part.
> 2. Just to keep the commit history consistent (no commit fails in the CI) move this commit to be the first in the series, as the CI bug you detected is independent from the current first commit.
> 3. I know that being a CI bug, you are limited in term of fixes, but is this fix "weak" (i.e., it may stop working in the future)? @rodrigosiqueira, what do you think?

Just made some changes in the pull request, but forgot 1. and 2.
I will change it in a moment again...